spanner_orm/field.py

```diff
@@ -59,26 +74,32 @@ def __init__(self,
                field_type: Type[FieldType],
                nullable: bool = False,
                primary_key: bool = False,
-               length: int = 0):
+               length: Optional[int] = None):
+    """Initialize Field class with given parameters.
+
+    Args:
+      field_type: The column field type.
+      nullable: Whether this field should be nullable. Default is false if not provided.
+      primary_key: Whether this field should be a primary key field. Default is false if not provided.
+      length: The length of this column field type if supported. Default is MAX if not provided.
```

Comment:
(nit) Don't duplicate the function signature, i.e., don't say what the default is.